src/test/resources/test-rom/spec/apis/colors_spec.lua

```diff
@@ -73,4 +73,39 @@ describe("The colors library", function()
         expect(colors.rgb8(0.3, 0.5, 0.6)):equals(0x4c7f99)
         expect({ colors.rgb8(0x4c7f99) }):same { 0x4c / 0xFF, 0x7f / 0xFF, 0.6 }
     end)
+
+    describe("colors.toBlit", function()
+        it("validates arguments", function()
+            expect.error(colors.toBlit, nil):eq("bad argument #1 (expected number, got nil)")
+        end)
+
+        it("converts all colors", function()
+            local colorsHex = {
```

Comment:
This version is a little shorter 😛.

```lua
local colorsHex = {}
for i = 0, 15 do colorsHex[2 ^ i] = string.format("%x", i) end
```

I guess it's less explicit, but I'm fine with that

Reply:
May as well remove the table entirely at that point